src/main/java/dataMgmt/MetadataManager.java

```diff
@@ -10,9 +10,7 @@
 import java.io.IOException;
 import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
```

Comment:
I don't think we have been preventing them [?]. If we do, we should go through the source and get rid of them all at once probably. If this is something we want to do, I can change this occurrence though.